components/IndividualRecipeCard.js

```diff
@@ -0,0 +1,60 @@
+import { useRouter } from 'next/router'; // Using instead of Link because problems that probably don't exist, could change back later...
+import Link from 'next/link';
+import { useEffect } from 'react';
+
+const IndividualRecipeCard = ({ recipe }) => {
+  const router = useRouter();
+
+  useEffect(() => {
```

Comment:
can you please add a screenshot to this PR so I can see what it looks like ? 

not familiar with "useEffect" (or any React stuff :D )  so will be helpful to visualize

Reply:
![image](https://user-images.githubusercontent.com/1176723/230986489-ba51f1e9-81fa-46b1-8158-8f68bd8c8711.png)

Here it is! It's not pretty (yet), but instead of having a recipe like the following, it makes it puts each on one line. Some how though the commas aren't actually getting stripped.

What it looked like before:
`Plant some limes, Pick some limes, Waaaash some limes, Roll some limes, Cut some limes, Squeeze some limes, Profit!`

Unsure if this will be 'better' on a mobile device, but rather than a large scroll, it's at least a bit more visually appealing on a desktop. Need to UX it a lot though.